returnn/tf/layers/basic.py

```diff
@@ -3909,15 +3909,21 @@ class PostfixInTimeLayer(_ConcatInputLayer):
 
   def __init__(self, postfix=0.0, repeat=1, **kwargs):
     """
-    :param float|int postfix: constant
+    :param float|int|LayerBase postfix: constant or other layer without time axis to use as postfix
     :param int repeat: how often to repeat the postfix
     """
     from returnn.tf.util.basic import DimensionTag
     super(PostfixInTimeLayer, self).__init__(**kwargs)
     self.output = self.input_data.copy(name="%s_output" % self.name)
     assert self.output.time_dim_axis is not None
-    assert isinstance(postfix, (float, int)), "other layer src not yet supported"
-    c = tf.constant(postfix, dtype=self.output.dtype)
+    assert isinstance(postfix, (float, int, LayerBase))
+    if isinstance(postfix, LayerBase):
+      assert not postfix.output.have_time_axis(), 'Postfix layer with time axis not implemented yet'
+      postfix = postfix.output.copy_compatible_to(self.output)
+      assert postfix.size_placeholder.get(self.output.time_dim_axis_excluding_batch, 1) == 1
```

Comment:
This check is wrong (if it is a tensor, `==` does not work like this). Just check instead:
```suggestion
      assert self.output.time_dim_axis_excluding_batch not in postfix.size_placeholder
```
Or maybe no need to check this at all.